### PR TITLE
Add basic file structure to the app

### LINKED ISSUE #3: Create basic file structure of the app
**Description**
The repository needs to be updated with the current progress f the chatbot (including all the code).

### PR DESCRIPTION
**Pull request by maayami**
Fix #3 
This PR adds the following file ->
- main.py
- Procfile
- runtime.txt
- requirements.txt
- example.env
- .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.venv/
+.vscode/
+__pycache__/
+*.py
+!main.py
+.env
```
Comment (rajitha1998) [inline .gitignore:6]: Keep a new line at the end of the file.

File: Procfile [hunk 2]
```diff
@@ -0,0 +1 @@
+worker: python main.py
```
Comment (rajitha1998) [inline Procfile:1]: Keep a new line at the end of the file.

File: main.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
+from gitterpy.client import GitterClient
```
Comment (rajitha1998) [inline main.py:1]: Please add comments for readability.

File: .gitignore (hunk 1 above)
Comment (maayami) [reply .gitignore:6]: Done

File: Procfile (hunk 2 above)
Comment (maayami) [reply Procfile:1]: Updated in the new commit.

File: main.py (hunk 3 above)
Comment (maayami) [reply main.py:2]: Done

**Comment by maayami** [comment]
@rajitha1998 Sir, I have added a new line at the end of every file but I think it's not visible in the Github changes.
Also, I have added comments in the main.py file to improve it's readability.

**Review by rajitha1998 — APPROVED** [review]

**Pull request merged**